docs/docs/Components/components-agents.md

```diff
@@ -21,310 +21,399 @@ The [simple agent starter project](/starter-projects-simple-agent) uses an [agen
+This component represents an Agent of CrewAI allowing for the creation of specialized AI agents with defined roles goals and capabilities within a crew.
 
-For more information, see the [CrewAI documentation](https://docs.crewai.com/how-to/Hierarchical/).
+For more information see the [CrewAI documentation](https://docs.crewai.com/core-concepts/Agents/).
 
-### Inputs
+<details>
+<summary>Parameters</summary>
+
+**Inputs**
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| agents | Agents | List of Agent objects representing the crew members |
-| tasks | Tasks | List of HierarchicalTask objects representing the tasks to be executed |
-| manager_llm | Manager LLM | Language model for the manager agent (optional) |
-| manager_agent | Manager Agent | Specific agent to act as the manager (optional) |
-| verbose | Verbose | Enables verbose output for detailed logging |
-| memory | Memory | Specifies the memory configuration for the crew |
-| use_cache | Use Cache | Enables caching of results |
-| max_rpm | Max RPM | Sets the maximum requests per minute |
-| share_crew | Share Crew | Determines if the crew information is shared among agents |
-| function_calling_llm | Function Calling LLM | Specifies the language model for function calling |
-
-### Outputs
+| role | Role | The role of the agent. |
+| goal | Goal | The objective of the agent. |
+| backstory | Backstory | The backstory of the agent. |
+| tools | Tools | The tools at the agent's disposal. |
+| llm | Language Model | The language model that will run the agent. |
+| memory | Memory | This determines whether the agent should have memory or not. |
+| verbose | Verbose | This enables verbose output. |
+| allow_delegation | Allow Delegation | This determines whether the agent is allowed to delegate tasks to other agents. |
+| allow_code_execution | Allow Code Execution | This determines whether the agent is allowed to execute code. |
+| kwargs | kwargs | Additional keyword arguments for the agent. |
+
+**Outputs**
 
 | Name | Display Name | Info |
 |------|--------------|------|
-| crew | Crew | The constructed Crew object with hierarchical task execution |
+| output | Agent | The constructed CrewAI Agent object. |
 
-## JSON Agent
+</details>
 
-This component creates a JSON agent from a JSON or YAML file and an LLM.
+### Hierarchical Crew
 
-### Inputs
+This component represents a group of agents managing how they should collaborate and the tasks they should perform in a hierarchical structure. This component allows for the creation of a crew with a manager overseeing the task execution.
 
-| Name | Type | Description |
-|------|------|-------------|
-| llm | LanguageModel | Language model to use for the agent |
-| path | File | Path to the JSON or YAML file |
+For more information see the [CrewAI documentation](https://docs.crewai.com/how-to/Hierarchical/).
```

Comment:
Per the Google style guide, use a comma after "For more information" - https://developers.google.com/style/cross-references#link-introductions